src/parallel_ops/fused_parallel_op.cc

```diff
@@ -281,7 +285,7 @@ size_t hash<FlexFlow::FusedParallelOpParams>::operator()(
     FlexFlow::FusedParallelOpParams const &params) const {
   size_t key = 0;
   hash_combine(key, params.parallel_ops.size());
-  for (ParallelOpInfo const &p : params.parallel_ops) {
+  for (auto const &p : params.parallel_ops) {
```

Comment:
Any reason for this change? I generally prefer to use explicit type names over `auto` when they're short just for clarity, but happy to be convinced otherwise :slightly_smiling_face:

Reply:
I like the ease of `auto` while writing code, but in this instance, I think I was actually just figuring out an issue with my linking as it wasn't picking up on the `ParallelOpInfo` import. I've reverted it back now since it's working.